workers/unity/Assets/Playground/Config/SystemConfig.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.PlayerLifecycle;
+using Improbable.Gdk.TransformSynchronization;
+using Unity.Entities;
+
+namespace Playground
+{
+    public static class SystemConfig
+    {
+        public const string UnityClient = "UnityClient";
+        public const string UnityGameLogic = "UnityGameLogic";
+
+        public static readonly Type[] SpatialSystems =
+        {
+            typeof(SpatialOSReceiveSystem),
+            typeof(SpatialOSSendSystem),
+            typeof(CleanReactiveComponentsSystem),
+            typeof(WorldCommandsSendSystem),
+            typeof(WorldCommandsCleanSystem),
+            typeof(CommandRequestTrackerSystem),
+        };
```

Comment:
Do we want to move this into `Core`?

Reply:
👍 could have something similar to the playerlifecycle & transform feature module